lib/logstash/filters/aggregate.rb

```diff
@@ -226,9 +248,15 @@ class LogStash::Filters::Aggregate < LogStash::Filters::Base
   # When this option is enabled, each time aggregate plugin detects a new task id, it pushes previous aggregate map as a new logstash event, 
   # and then creates a new empty map for the next task.
   # 
+  # When the timeout code is set, the event populated with the aggregation map is passed to the timeout code before the event is yielded
```

Comment:
Up to me, this sentence is not at the right place. It is more relevant in "timeout_code".

Especially as we can use push_previous_map_as_event option without using "timeout_code" if a simple aggregation map copy is enough.